Libraries/LibC/unistd.h

```diff
@@ -72,7 +72,7 @@ int execlp(const char* filename, const char* arg, ...);
 int chroot(const char* path);
 int chroot_with_mount_flags(const char* path, int mount_flags);
 void sync();
-void _exit(int status);
+__attribute__((noreturn)) void _exit(int status);
```

Comment:
Let's use the standard style `[[noreturn]]` :)

Reply:
This is a C header.

Reply:
Oops!